app/javascript/blacklight/modal.js

```diff
@@ -168,8 +168,11 @@ const Modal = (() => {
     // Register both trigger and preserve selectors in ONE event handler, combining
     // into one selector with a comma, so if something matches BOTH selectors, it
     // still only gets the event handler called once.
-    $('body').on('click', modal.triggerLinkSelector + ', ' + modal.preserveLinkSelector,
-      Blacklight.modal.modalAjaxLinkClick);
+    document.addEventListener('click', (e) => {
+      if (e.target.matches(`${modal.triggerLinkSelector}, ${modal.preserveLinkSelector}`))
+        modal.modalAjaxLinkClick(e)
```

Comment:
This is pretty cool.  I did not know you could interpolate variables like this in JavaScript.